system/lib/standalone/standalone.c

```diff
@@ -34,8 +34,9 @@
 
 // libc
 
-void abort() {
-  _Exit(1);
+void _abort_js(void) {
+  __builtin_trap();
```

Comment:
Why did this change? And if it is correct, why is there anything after it?

Reply:
Yes, I think this is correct.

I'm mostly copying from musl where it looks like the `_Exit()` call is there as double check: https://github.com/emscripten-core/emscripten/blob/d9aed286296e04b80f05b6f827d7434d1a3e115f/system/lib/libc/musl/src/exit/abort.c#L26-L29|

I will duplicate the comment here so it clear.